permissions/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Wed Sep 25 17:23:11 CEST 2019
```

Comment:
I suspect we don't need the wrapper here.

Reply:
I'll remove whole folder for module